### ISSUE TITLE
AVoiceOver Class is Broken

### ISSUE THREAD

**Issue by tbreitenfeldt** [accessibleapps/accessible_output2#8]
The VoiceOver class for Mac does not work any longer. An error is thrown when ran. Digging showed that the AppScript library used to communicate with VoiceOver is no longer supported.
This issue has been talked about at length on the audiogames.net form. The thread to this discussion is below:

https://forum.audiogames.net/topic/37573/voiceover-error-using-accessibleoutput2/

A proposed solution is suggested, rolling back to the older way of communicating to VoiceOver using apple script. although an issue was discovered with this proposed solution where about a 2 second delay occurs when is_active is called. Still unsure about how to go about fixing this issue.

An alternative using a library called AppKit was suggested, although never investigated as a solution. See here for the links mentioned in the audiogames thread:

https://developer.apple.com/documentati … ynthesizer

https://python.hotexamples.com/examples … mples.html

https://stackoverflow.com/questions/472 … med-appkit
